src/components/Landing/QAs.tsx

```diff
@@ -1,4 +1,19 @@
-import { t } from '@lingui/macro'
+import { t, Trans } from '@lingui/macro'
+
+export const OverflowVideoLink = ({ text }: { text: string }) => (
```

Comment:
**note (non-blokcing**) we can probably just have a single `ExternalLink` component in the future that sets target and rel attributes. Future improvement!